test/test_board.py

```diff
@@ -412,27 +412,6 @@ def test_place_holes(self):
             ]
             self.assertEqual(sorted(holes_positions), sorted(holes))
 
-    def test_holes_quantity(self):
```

Comment:
This test failed because with the _valid_hole() method modifications the patched_game() gives unexpected results.
The test was replaced by test_only_right_HOLE_QUANTITY_have_to_be_placed()